src/views/index.html

```diff
@@ -21,6 +21,13 @@
     </div>
     <div class="landing-welcome-content">
       <img class="landing-welcome-logo" src="{{ asset('images/logos/hc2017-logo-full-light.svg') }}" alt="Hack Cambridge: Recurse">
+      {# Co-host #}
+      <p>
+        &nbsp; &nbsp; &nbsp; &nbsp; Made possible by <br>
```

Comment:
It's weird that we need these non-breaking spaces, but it definitely looks better with them. Would it be more predictable to push the text with margin? Depending on the width of spaces of our font seems flaky to me.